.github/workflows/checks.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         uses: github/super-linter@v4
         env:
           VALIDATE_ALL_CODEBASE: false
-          DEFAULT_BRANCH: ${{ github.ref_name }}
+          DEFAULT_BRANCH: master
```

Comment:
Checks doesn't work if this is set so have you done further testing against this?

Reply:
It actually does! Check [my current MediaPipe.NET PR](https://github.com/vignetteapp/MediaPipe.NET/pull/1) where it actually tried and succeeds in linting.

The docs said that this is supposed to be the default branch of the repository (and not the branch being linted), so it would make sense for it to be `master`.